### PR TITLE
Ato 82/modify stubs to emulate post requests to authorize

### PR DESCRIPTION
**Pull request by gn-GDS**
## What?

Add ability for the auth stub to perform POST requests.

## Why?

GET requests have a string limit.
We currently don’t know what our current string limit is or if it’s a problem. 
This will remove risk of hitting query string limits with the full encrypted and signed payload

## Related PRs
https://govukverify.atlassian.net/browse/ATO-22
https://govukverify.atlassian.net/browse/ATO-81
https://govukverify.atlassian.net/browse/ATO-88

### CODE REVIEW

**Comment by kerrr** [comment]
I think the unsigned commits might cause an issue?

**Comment by CarlyG55** [comment]
> I think the unsigned commits might cause an issue?
Ah good spot, thanks! I unsuccessfully tried to move from GPG to SSH signing haha

**Review by ethanmills — DISMISSED** [review]

File: README.MD [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+To run the stub locally, and make calls to sandpit (defined by `OP_BASE_URL`) you can run:
+```shell
+export OP_BASE_URL="https://oidc.sandpit.account.gov.uk/";
+./startup.sh
+```
+
+## Deploy To Sandpit
+**This is due to be moved to aws. It is difficult to deploy as PaaS is EOL.**
+
+To deploy to the sandpit enviroment, you will need access to the PaaS service (see [here](https://docs.cloud.service.gov.uk/get_started.html#enable-single-sign-on)).
```
Comment (ethanmills) [inline README.MD:18]:
```suggestion
To deploy to the sandpit enviroment, you will need access to the PaaS service. You should then enable SSO (see [here](https://docs.cloud.service.gov.uk/get_started.html#enable-single-sign-on)).
```

File: README.MD [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+
+You also need cloud foundry (cf) locally setup.
+`brew install cloudfoundry/tap/cf-cli@8`
+
+You first login using:
+```
+cf login -a api.london.cloud.service.gov.uk -u [name here]@digital.cabinet-office.gov.uk
```
Comment (ethanmills) [inline README.MD:25]:
```suggestion
cf login -a api.london.cloud.service.gov.uk --sso
```

**Review by ethanmills — APPROVED** [review]

**Pull request merged**